ERCS/erc-7776.md

```diff
@@ -0,0 +1,554 @@
+Earnings Per Token (E.P.Share-Token), allowing investors to assess the financial health of blockchain-based companies in a standardized, transparent, clear and reliable format.
+
+## Motivation
+
+The motivation of this E.I.P. is to Bring Seriousness to the CryptoCurrencies Investments Market. Currently, the situation is as follows:
+
+Most [ERC-20](./eip-20.md) Tokens representing E.V.M. Blockchain-based companies (or also called "Protocols"), DO NOT work the same way as a Publicly Traded Stock that represents a Share of ownership of the equity of that such company (so the user who buys a Protocol's ERC-20, is also now a share-holder and co-owner of the business, its profits and/or its dividends), but rather function as "Commodities" such as oil; they are consumable items created by said E.V.M. Blockchain-based company (or "Protocol") to be spent in their platform. They are Publicly Traded and advertised to be representing the underlying Protocol like a Share, working in practice the same way as a Commodity and without any Public, Transparent and _Clear_ Financial Information as publicly traded stocks have.
+
+Added to that, most token research analysis reports that can be currently found on the internet are informal Substack or Twitter posts, with lots of abstract explanations about the features of the said Protocol to invest in, that lack of transparent financial numbers and factual financial information, that are made by anonymous users without real exposed reputations to affect.
+
+This E.I.P. will improve that by giving users and investors transparent, clear and factual financial information to work with when analyzing as a potential investment the such
+E.V.M. Blockchain-based company that implements this E.I.P. in their Solidity Smart Contracts, and that will generate Trust, Transparency and Seriousness in the CryptoCurrencies Investments Market long term.
+
+## Specification
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119 and RFC 8174.
+
+All Transparent Financial Statements Standard implementations MUST implement ERC-20 to represent shares, and the financial numbers displayed in a StableCoin's value.
+
+All Transparent Financial Statements MUST implement ERC-20's optional metadata extensions.
+The `name` and `symbol` functions SHOULD reflect the underlying token's `name` and `symbol` in some way.
+
+All methods MUST be of visibility `public`.
```

Comment:
Why not external?

Reply:
Good question🤔. To be honest, I never thought about it. But since this information is meant to be consumed mainly by the public, it makes a little more sense for it to be `external`. Changed.